right-click-for-superpowers/script.js

```diff
@@ -0,0 +1,117 @@
+import {FilesetResolver, LlmInference} from 'https://cdn.jsdelivr.net/npm/@mediapipe/tasks-genai';
+
+const MODEL_FILE_NAME = 'https://storage.googleapis.com/jmstore/kaggleweb/grader/g-2b-it-gpu-int4.bin';
```

Comment:
I'm not sure we want to check in this url. Maybe leave a placeholder for the user to fill in, like in this demo: https://github.com/GoogleChromeLabs/web-ai-demos/blob/main/product-reviews/.env.template

You can add a note in Readme that developer needs to provide path/url to the downloaded model.

Reply:
Done